letsencrypt/config.json

```diff
@@ -9,19 +9,79 @@
   "boot": "manual",
   "ports": {
     "80/tcp": 80
-   },
+            },
   "map": ["ssl:rw"],
   "options": {
-    "email": null,
-    "domains": [null],
+    "email": "your@email.com",
+    "domains": ["your.domain"],
     "certfile": "fullchain.pem",
-    "keyfile": "privkey.pem"
+    "keyfile": "privkey.pem",
+    "challenge": "http",
+    "dnsprovider": "",
+    "dns_cloudflare_email": "your@email.com",
+    "dns_cloudflare_api_key": "",
+    "dns_cloudxns_api_key": "",
+    "dns_cloudxns_secret_key": "",
+    "dns_digitalocean_token": "",
+    "dns_dnsimple_token": "",
+    "dns_dnsmadeeasy_api_key": "",
+    "dns_dnsmadeeasy_secret_key": "",
+    "dns_gehirn_api_token": "",
+    "dns_gehirn_api_secret": "",
+    "dns_linode_key": "",
+    "dns_linode_version": "",
+    "dns_luadns_email": "your@email.com",
+    "dns_luadns_token": "",
+    "dns_nsone_api_key": "",
+    "dns_ovh_endpoint": "",
+    "dns_ovh_application_key": "",
+    "dns_ovh_application_secret": "",
+    "dns_ovh_consumer_key": "",
+    "dns_rfc2136_server": "",
+    "dns_rfc2136_port": "",
+    "dns_rfc2136_name": "",
+    "dns_rfc2136_secret": "",
+    "dns_rfc2136_algorithm": "",
+    "aws_access_key_id": "",
+    "aws_secret_access_key": "",
+    "dns_sakuracloud_api_token": "",
+    "dns_sakuracloud_api_secret": ""
   },
   "schema": {
     "email": "email",
     "domains": ["str"],
     "certfile": "str",
-    "keyfile": "str"
+    "keyfile": "str",
+    "challenge": "str",
+    "dnsprovider": "str",
+    "dns_cloudflare_email": "email",
```

Comment:
```
"dns": {
   "cloudflare_email": "",
   ....
}
```

Reply:
Not sure what you are looking for in this one.

Reply:
Unclear why to further segment?

Reply:
Because we have soon an UI support and want group that on UI differently from root

Reply:
Because we have soon an UI support and want group that on UI differently from root